src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key in action.keysToRemove) {
+          if (state[action.keysToRemove[key]]) {
+            delete state[action.keysToRemove[key]];
```

Comment:
```suggestion
            delete state[key];
```
Якщо ти використаєш цикл for of, то можна буде просто звертатись тут по ключу